docs/admin/clustering/scale/demand.md

```diff
@@ -0,0 +1,188 @@
+## Introduction
+
+Many organizations size their database infrastructure to handle the maximum level of load they can anticipate, but very often load is seasonal, in some cases around specific events on certain days of the year. Compromises are often made where infrastructure sits idle most of the year and performance is not as good as desired when requests peak.
+
+CrateDB allows clusters to be scaled both up and down by adding and removing nodes, this allows significant savings during quiet periods, and also the provisioning of extra capacity during particular periods of high activity to have optimal performance.
+
+When nodes are added or removed CrateDB automatically rebalances shards, but in cases where we have very large volumes of historical data and new nodes are only added for a short period of time, we may want to avoid any of the historical data being relocated to the temporary nodes.
+
+## About
+The approach described below explains how to achieve this using [shard allocation filtering](https://crate.io/docs/crate/reference/en/5.1/general/ddl/shard-allocation.html) on a table that is partitioned by day, the idea is that the period of high activity can be foreseen, so the scaling up will take place the day before a big event, and the scaling down someday after the event has ended.
+
+This same approach can be applied to multiple tables. It is particularly relevant for the larger tables, and smaller tables can be kept on the baseline nodes, but it is always good to consider the impact on querying performance if the small tables will be queried during the big event `JOIN`ed to the big tables that will have data on the temporary nodes.
+
+## Preparing the test environment
+
+In this example, we will imagine that the surge in demand we are preparing for is related to the 2022 FIFA Men’s World Cup running from 20/11/2022 to 18/12/2022.
+
+We will start with a 3 nodes cluster, on which we will create a test table and populate it with some pre-World Cup data:
+
+```sql
+CREATE TABLE test (
+  ts TIMESTAMP,
+  recorddetails TEXT,
+  "day" GENERATED ALWAYS AS date_trunc('day',ts)
+  )
+PARTITIONED BY ("day")
+CLUSTERED  INTO 4 SHARDS
+WITH (number_of_replicas=1);
+
+INSERT INTO test (ts) VALUES ('2022-11-18'),('2022-11-19');
+```
+
+The shards will initially look like this:
+![image|690x167](https://global.discourse-cdn.com/flex020/uploads/crate/original/1X/ac18a9cb507201d8e54771e320501f4aaac0eb16.png)
+
+## Before deploying extra nodes
+
+We want to make sure that the addition of the temporary nodes does not result on data from the large tables getting rebalanced to use these nodes.
+
+We will be able to identify the new nodes by using a custom attribute (`node.attr.storage=temporarynodes`) (see further down for details on how to configure this), so the first step is to configure the existing partitions so that they do not consider the new nodes as suitable targets for shard allocation.
+
+In CrateDB 5.1.2 or higher we can achieve this with:
```

Comment:
I haven't tested it yet, but since [5.9.0](https://cratedb.com/docs/crate/reference/en/master/appendices/release-notes/5.9.0.html), it shouldn't be needed anymore to change allocation for every single table, but default allocation rules can be set on the cluster level:
> Added support to override routing.allocation.* cluster settings with a routing.allocation.* table setting. This can be used to define the default routing behavior for all tables with a cluster setting and reroute individual tables by assigning the table setting using ALTER TABLE SET.

Reply:
Hi. 8f9a86ca998 adds a relevant note.